.github/workflows/ci-release-gate.yml

```
@@ -242,13 +244,13 @@ jobs:
       dockerhub_token: ${{ secrets.DOCKER_HUB_ACCESS_TOKEN }}
       ghcr_username: ${{ github.repository_owner }}
       ghcr_token: ${{ secrets.GITHUB_TOKEN }}
-      ADMIN_PASS: ${{ vars.ADMIN_PASS }}
-      DB_PASS: ${{ vars.DB_PASS }}
-      DB_USER: ${{ vars.DB_USER }}
-      DB_NAME: ${{ vars.DB_NAME }}
-      JWT_SECRET: ${{ secrets.JWT_SECRET }}
-      DATA_ENCRYPTION_KEY: ${{ secrets.DATA_ENCRYPTION_KEY }}
-      ADMIN_USER: ${{ vars.ADMIN_USER }}
+      admin_pass: ${{ vars.ADMIN_PASS }}
+      db_pass: ${{ vars.DB_PASS }}
```

actionlint (1.7.12)

[error] secret "db_pass" is not defined in "./.github/workflows/dast-release.yml" reusable workflow. defined secrets are "admin_pass", "admin_user", "data_encryption", "db_name", "db_user", "dockerhub_token", "dockerhub_username", "eb_pass", "ghcr_token", "ghcr_username", "jwt_secret"

(workflow-call)